dlib/CMakeLists.txt

```diff
@@ -24,12 +24,15 @@ set(CPACK_PACKAGE_VERSION_MINOR "8")
 set(CPACK_PACKAGE_VERSION_PATCH "0")
 set(VERSION ${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH})
 # Set DLIB_VERSION in the including CMake file so they can use it to do whatever they want. 
-get_directory_property(has_parent PARENT_DIRECTORY)
-if(has_parent)
-   set(DLIB_VERSION ${VERSION} PARENT_SCOPE)
-   if (NOT DEFINED DLIB_IN_PROJECT_BUILD)
+
+if(NOT HUNTER_ENABLED)
```

Comment:
Would it be clearer if original code remains the same but after this block we will do:
```cmake
if(HUNTER_ENABLED)
  set(DLIB_IN_PROJECT_BUILD false)
endif()
```
?

Reply:
Yes.